UI_UX/Frontend/src/styles/Achievements.scss

```diff
@@ -0,0 +1,80 @@
+.achievements {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  padding-top: 70px;
+  padding-bottom: 70px;
+  padding-left: 75px;
+  overflow-y: scroll;
+
+  &> li {
```

Comment:
Descendent selectors are actually not that optimal from, and we don't do element selectors. A better way to do this would be:
`&__list-item`